website/src/commands/append.md

```diff
@@ -36,23 +36,23 @@ main
 *   feature-2
 ```
 
-### --detached / -d
+#### `-d`<br>`--detached`
```

Comment:
Thanks for pointing this out. I changed this with a regex that matched `### -`. I'll review the docs for any other missed headings that need a heading level change.

FYI, I changed the heading level because I think the smaller font looks better when formatted as code, especially for multi-line headings.

Reply:
You clearly have better taste and more talent in this area than I do. So I'm happy to leave such choices up to you.